### DISCUSSION TITLE
Drasil Research Group Meeting Agenda, Friday Jan 28, 1:30 pm, 2022 (On Teams)

### DISCUSSION THREAD

**Discussion by smiths** [JacquesCarette/Drasil#2925] [Meeting Agendas]
The agenda for this coming week is as follows:

1. Dong update
2. Jason update
3. Peter update
4. Feedback on Refining Theories version of SRS for BT:consThermE to RT:rocTempSimp
[Theories version of noPCM](https://github.com/smiths/caseStudies/blob/master/CaseStudies/noPCM/docs/SRS/TheoriesVersion_NoPCM_SRS.pdf) 
[Diagram of Refinement](https://github.com/smiths/caseStudies/blob/master/CaseStudies/noPCM/docs/SRS/TheoryRefinement_NoPCM.drawio.pdf)
5. Cold cases:
a. Follow up to #1166 (Peter)
b. #2911 (Jason)
c. #2194 (Don)
d. #412 - At the very least, if we choose to not use pandoc, we might be able to learn from some of its [inner mechanics](https://hackage.haskell.org/package/pandoc-types-1.22.1/docs/src/Text.Pandoc.Definition.html#Pandoc)  (Jason)
e. #1715 (Peter)
6. Questions/Comments/Concerns/Discussion

**Comment by balacij** [comment]
[Meeting recording](https://mcmasteru365.sharepoint.com/sites/SummerDrasil/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FSummerDrasil%2FShared%20Documents%2FGeneral%2FRecordings%2FDrasil%20Research%20Meeting%2D20220128%5F134048%2DMeeting%20Recording%2Emp4&parent=%2Fsites%2FSummerDrasil%2FShared%20Documents%2FGeneral%2FRecordings) (internal to McMaster students, professors, etc)
